src/core/api/abuse.js

```diff
@@ -20,3 +19,20 @@ export function reportAddon(
     state: api,
   });
 }
+
+export type ReportUserParams = {|
+  api: ApiStateType,
+  message: string,
+  user: UserType,
+|};
+
+export function reportUser({ api, message, user }: ReportUserParams) {
```

Comment:
I'd like to see the return type defined, like `: Promise<ReportUserReponse>` because it would be helpful to know how the saga should deal with it.